LayoutTests/accessibility/color-well-legacy.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN"><!-- webkit-test-runner [ InputTypeColorEnhancementsEnabled=false ] -->
+<html>
+<head>
+<script src="../resources/js-test.js"></script>
+<script src="../resources/accessibility-helper.js"></script>
+</head>
+<body>
+
+<input id="empty_colorwell" type="color">
+<input id="good_colorwell" type="color" value="#ff0000">
+<input id="bad_colorwell" type="color" value="purple">
+
+<script>
+    description("This test checks the role of ColorWellRole on an input with type=color");
```

Comment:
In newer accessibility tests, we aren't indenting a level inside a `<script>` tag, so everything inside can move back one tab.